CONTRIBUTING.md

```diff
@@ -3,6 +3,13 @@
 We'd love to accept your patches and contributions to this project. There are
 just a few small guidelines you need to follow.
 
+## How to build
```

Comment:
I would say,

## Building

Currently, druid only builds on windows and macOS.

### macOS

On macOS, druid requires cairo; see [gtk-rs dependencies] for installation instructions.
You may also need to set your `PKG_CONFIG_PATH`; assuming you have installed `gtk` through homebrew,
you can builld with,

```sh
$> PKG_CONFIG_PATH="/usr/local/opt/libffi/lib/pkgconfig" cargo build
```

Reply:
I'm a little confused about the build instructions. My understanding is that we're asking people to install cairo on the mac build (ie `brew install cairo`), not all of gtk. Just cairo is a considerably lighter weight dependency.

Reply:
Updated.

Reply:
I think we are mixing target platform and building platform.
Brew is only available in MacOS. But any platfrom, trying to build macos targeted binaries has to have hits dependencies. 
So we cannot just ask here to use homebrew.

Reply:
Are you talking about cross-compiling? I don't think we're supporting that, I can think of a number of ways in which that would be tricky, more so going forward (we may well want to depend on xcode-specific tools to, for example, packresources into an app bundle).

Reply:
Well, at least right now I can build for both platforms. But let's not support it officially until tested properly. I will update PR soon.

Reply:
Readme updated